subsys/bluetooth/host/classic/br.c

```diff
@@ -141,7 +141,7 @@ static bool br_sufficient_key_size(struct bt_conn *conn)
 		return key_size == BT_HCI_ENCRYPTION_KEY_SIZE_MAX;
 	}
 
-	return key_size >= BT_HCI_ENCRYPTION_KEY_SIZE_MIN;
+	return key_size >= CONFIG_BT_SMP_MIN_ENC_KEY_SIZE;
```

Comment:
I guess it would be good to explicitly document somewhere that in Zephyr you can't disable LE, and that the BR/EDR key size requirement is directly inherited from the LE one (for SMP).

Reply:
Currently, `BT_SMP` is selected by `BT_CLASSIC` in Bluetooth classic Kconfig.

Reply:
> Currently, `BT_SMP` is selected by `BT_CLASSIC` in Bluetooth classic Kconfig.

Maybe it'd make sense to add to the help text of `BT_SMP_MIN_ENC_KEY_SIZE` that it will also be used for Classic in case that's supported & enabled?

Reply:
> > Currently, `BT_SMP` is selected by `BT_CLASSIC` in Bluetooth classic Kconfig.
> 
> Maybe it'd make sense to add to the help text of `BT_SMP_MIN_ENC_KEY_SIZE` that it will also be used for Classic in case that's supported & enabled?

Alternatively (and maybe better/cleaner), create a new Classic-specific Kconfig option that has `default BT_SMP_MIN_ENC_KEY_SIZE`

Reply:
Updated.